content/en/apps/guides/hosting/4.x/data-migration.md

```diff
@@ -195,18 +195,18 @@ EOF
 
 h) Check that `couchdb-migration` can connect to the CouchDb instance and that CouchDb is running. You'll know it is working when the `docker-compose` call exits without errors and logs `CouchDb Cluster is Ready`.
 ```shell
-cd ~/couchdb-migration/ 
+cd ~/couchdb-migration/
 docker-compose run couch-migration check-couchdb-up <number-of-nodes>
 ```
 
-i) Generate the shard distribution matrix and get instructions for final shard locations. 
+i) Generate the shard distribution matrix and get instructions for final shard locations.
 ```shell
-cd ~/couchdb-migration/ 
+cd ~/couchdb-migration/
 shard_matrix=$(docker-compose run couch-migration generate-shard-distribution-matrix)
 docker-compose run couch-migration shard-move-instructions $shard_matrix
-``` 
+```
 
-j) Follow the instructions from the step above and move the shard files to the correct location, according to the shard distribution matrix. 
+j) Follow the instructions from the step above and move the shard files to the correct location, according to the shard distribution matrix. This is a manual step that requires to physically move data around on disk.
```

Comment:
My recommendation is to use `rsync` with the flags `-avz --progress`. The `-a` ensures that symbolic links, devices, attributes, permissions, ownership, etc., are preserved in the copied files. The `v` just for verbose mode and the `z` to send the files in a compressed format to preserve bandwidth. The `--progress` is good because you often wonder during file transfers if things are really working or if it's stuck somewhere.

Reply:
I think this will heavily depend on deployment. Some deployments hold all data locally,  maybe rsync doesn't make sense for those.